modnet/preprocessing.py

```diff
@@ -584,7 +572,12 @@ def featurize(self, fast: bool = False, db_file: str = 'feature_database.pkl', n
         self.df_featurized = df_final
         logging.info('Data has successfully been featurized!')
 
-    def feature_selection(self, n: int = 1500, cross_nmi: Optional[pd.DataFrame] = None):
+    def feature_selection(
+        self,
+        n: int = 1500,
+        cross_nmi: Optional[pd.DataFrame] = None,
+        use_precomputed_cross_nmi: bool = False
```

Comment:
Wouldn't it make more sense to keep the current default behaviour (using the MP cross NMI), and set us_precomputed_cross_nmi to True as default ?